interface/wx/dcsvg.h

```diff
@@ -21,24 +21,30 @@
     is a write-only class.
 
     As the wxSVGFileDC is a vector format, raster operations like GetPixel()
-    are unlikely to be supported. However, the SVG specification allows for PNG
-    format raster files to be embedded in the SVG, and so bitmaps, icons and
-    blit operations in wxSVGFileDC are supported.
+    are unlikely to be supported. However, the SVG specification allows for
+    raster files to be embedded in the SVG, and so bitmaps, icons and blit
+    operations in wxSVGFileDC are supported. By default only PNG format bitmaps
+    are supported and these are saved as separate files in the same folder
+    as the SVG file, however it is possible to change this behaviour by
+    replacing the built in bitmap handler using wxSVGFileDC::SetBitmapHandler().
 
     A more substantial SVG library (for reading and writing) is available at
     the wxArt2D website <http://wxart2d.sourceforge.net/>.
 
     @library{wxcore}
     @category{dc}
 */
+
 class wxSVGFileDC : public wxDC
 {
 public:
     /**
         Initializes a wxSVGFileDC with the given @a f filename with the given
-        @a Width and @a Height at @a dpi resolution.
+        @a Width and @a Height at @a dpi resolution, and an optional @a title.
+        The title provides a readable name for the SVG document.
```

Comment:
Should document title as new since 3.0.5.

Reply:
I will not back-port this feature.